decode/radio_frame.js

```diff
@@ -1,45 +1,43 @@
 var EthernetAddr = require('./ethernet_addr');
 var LogicalLinkControl = require('./llc_packet');
-
+var RadioBeaconFrame = require('./radio_beacon_frame');
 function RadioFrame() {
```

Comment:
Here is a jsperf for your gist: http://jsperf.com/constructorvsmethod

Personally I don't think there is a big readability difference. However, when in rome. Will make the change.